src/pudl/transform/ferc1.py

```diff
@@ -2957,19 +2959,56 @@ def merge_xbrl_metadata(
         return df
 
 
-class AccumulatedDepreciationPlantFerc1TableTransformer(Ferc1AbstractTableTransformer):
-    """Transformer class for :ref:`accumulated_depreciation_plant_ferc1` table."""
+class ElectricPlantDepreciationChangesFerc1TableTransformer(
+    Ferc1AbstractTableTransformer
+):
+    """Transformer class for :ref:`electric_plant_depreciation_changes_ferc1` table."""
 
-    table_id: TableIdFerc1 = TableIdFerc1.ACCUMULATED_DEPRECIATION_PLANT_FERC1
+    table_id: TableIdFerc1 = TableIdFerc1.ELECTRIC_PLANT_DEPRECIATION_CHANGES_FERC1
     has_unique_record_ids: bool = False
 
+    @cache_df("process_xbrl_metadata")
+    def process_xbrl_metadata(self, xbrl_metadata_json) -> pd.DataFrame:
+        """Transform the metadata to reflect the transformed data.
+
+        Replace the name of the balance column reported in the XBRL Instant table with
+        starting_balance / ending_balance since we pull those two values into their own
+        separate labeled rows, each of which should get the original metadata for the
+        Instant column.
+        """
+        meta = (
+            super()
+            .process_xbrl_metadata(xbrl_metadata_json)
+            .assign(
+                xbrl_factoid=lambda x: x.xbrl_factoid.replace(
+                    {
+                        "accumulated_provision_for_depreciation_of_electric_utility_plant": "starting_balance"
+                    }
+                )
+            )
+        )
+        ending_balance = meta[meta.xbrl_factoid == "starting_balance"].assign(
+            xbrl_factoid="ending_balance"
+        )
```

Comment:
so you are renaming the original xbrl label as start bal, then effectively duplicating that record with all of its attributes and calling it ending balance because we need it twice. yeah that makes sense for this table imo

Reply:
It's not pretty, but that seemed to be what was needed for consistency.